packages/marko-web-inquiry/browser/default-form.vue

```diff
@@ -156,6 +165,10 @@ export default {
       type: String,
       default: null,
     },
+    sitekey: {
+      type: String,
+      default: null,
```

Comment:
```suggestion
      required: true,
```

Make the sitekey required, as you're using it below.